makeflow/src/makeflow.c

```diff
@@ -1233,6 +1233,7 @@ int main(int argc, char *argv[])
 
 	random_init();
 	debug_config(argv[0]);
+	debug_config_file_size(string_metric_parse("0"));//to set debug file size to "don't delete anything"
```

Comment:
Is there any need to pass this through `string_metric_parse`? I believe you can either create an off_t or just pass 0, with no need to added a string conversion into the mix.